### PR TITLE
PR for [Compute API] Verify values for the body parameters runtime and provider in api/v<VERSION>/defenders/serverless/bundle #39747

### PR DESCRIPTION
**Pull request by ssugandh**
Work is in progress. Not ready to merge.

### CODE REVIEW

**Comment by Pubs-MV** [comment]
@ssugandh please remove the label 'do-not-merge' when you're ready.

File: api/descriptions/defenders/serverless-bundle_post.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 Downloads a ZIP file with serverless Defender bundle.
 
+Refer to the following values to assign:
+* For `provider`: Use `aws` or `azure`.
+* For `runtime`: 
+    * If you use `aws`, then you can add the runtime values: `python`, `python3.6`, `python3.7`, `python3.8`, `python3.9`, `nodejs12.x`, `nodejs14.x`, `dotnet6`, `java8`, `java11`, or `ruby2.7`.
+    * If you use `azure`, then you can add `dotnet6`.
```
Comment (Elichy) [inline api/descriptions/defenders/serverless-bundle_post.md]: dotnet6, dotnet3

Comment (ssugandh) [reply api/descriptions/defenders/serverless-bundle_post.md]: For both aws and azure or only azure?

Comment (Elichy) [reply api/descriptions/defenders/serverless-bundle_post.md]: Only azure.

Comment (ssugandh) [reply api/descriptions/defenders/serverless-bundle_post.md]: Thanks, I corrected. Would you be able to approve this PR?

**Review by Elichy — APPROVED** [review]

**Review by Pubs-MV — APPROVED** [review]

**Pull request merged**